tests/kuttl/tests/dataplane-deploy-global-service-test/00-assert.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNodeSet
+metadata:
+  name: edpm-compute-global-service
+  namespace: openstack
+spec:
+  preProvisioned: true
+  services:
+  - download-cache
+  - bootstrap
+  - configure-network
+  - validate-network
+  - install-os
+  - configure-os
+  - run-os
+  - install-certs
+  - ovn
+  - neutron-metadata
+  - neutron-ovn
+  - neutron-sriov
+  - neutron-dhcp
+  - libvirt
+  - nova
```

Comment:
you're missing `custom-global-service` here

Reply:
I went ahead and added it and pushed to your branch.

Reply:
Squashed in original, thank you James.